doc/apidoc/Makefile

```diff
@@ -21,4 +21,8 @@ help:
 	sphinx-apidoc -o generated ../../packages/python/plotly/plotly ../../packages/python/plotly/plotly/validators ../../packages/python/plotly/plotly/tests ../../packages/python/plotly/plotly/matplotlylib/ ../../packages/python/plotly/plotly/offline ../../packages/python/plotly/plotly/api
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 	rename 's/graph_objs/graph_objects/' _build/html/*.html _build/html/generated/*.html
-	sed -i 's/graph_objs/graph_objects/g' _build/html/*.html _build/html/generated/*.html
+	sed -i 's/graph_objs/graph_objects/g' _build/html/*.html 
```

Comment:
could we just have one big `sed` command with `**/**` or something that would catch all these cases?

Reply:
oh yes probably I just got confused with the syntax. Will try again.